servers/rendering/rendering_device.cpp

```diff
@@ -2229,6 +2230,9 @@ RDD::RenderPassID RenderingDevice::_render_pass_create(RenderingDeviceDriver *p_
 	for (int i = 0; i < p_passes.size(); i++) {
 		const FramebufferPass *pass = &p_passes[i];
 		RDD::Subpass &subpass = subpasses[i];
+		subpass.color_references.reserve(pass->color_attachments.size());
```

Comment:
Are you sure the subpass lists are empty at this point?

Reply:
Yes, the whole `subpasses` is freshly defined just 7 lines above.